terraform/modules/emr/templates/emr/poll_status_table.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+if [[ -n "$1" ]]; then
+  DATASOURCE="$1"
+else DATASOURCE="PDM"
+fi
+
+if [[ -n "$2" ]]; then
+  TIMEOUT="$2"
+else
+  TIMEOUT=3600
+fi
+
+((RETRIES=$TIMEOUT/30))
+
+count=0
+query="SELECT 1 FROM audit.data_pipeline_metadata_hive WHERE dataproduct = '${DATASOURCE}' AND upper(status) = 'COMPLETED' AND dateproductrun = current_date();"
+
+while [ $count -lt $RETRIES ]; do
+  status=$(hive -S -e "${query}")
```

Comment:
Can we add some output to the script in terms of what the parameters we are using are set to, and what the current retry count is, so we can see that information in the azkaban logs.